VAMobile/src/components/Nametag/Nametag.tsx

```diff
@@ -39,22 +54,51 @@ export const Nametag: FC = () => {
     }
   }
 
+  const pressableProps: PressableProps = {
+    onPress: accessToMilitaryInfo && showVeteranStatus ? navigateTo('VeteranStatus') : undefined,
+    accessibilityRole: accessToMilitaryInfo ? 'button' : undefined,
+    accessibilityLabel: accessToMilitaryInfo ? `${name()} ${branch} ${t('veteranStatus.title')}` : undefined,
+  }
+
   return (
-    <Box width="100%" backgroundColor="profileBanner" minHeight={85} display="flex" justifyContent="center" mb={theme.dimensions.standardMarginBetween} accessible={true}>
-      <Box py={accessToMilitaryInfo ? theme.dimensions.cardPadding : 0} display="flex" flexDirection="row">
-        {accessToMilitaryInfo && <Box pl={theme.dimensions.cardPadding}>{getBranchSeal()}</Box>}
-        <Box ml={20} flex={1}>
-          <TextView textTransform="capitalize" mb={theme.dimensions.textIconMargin} variant="BitterBoldHeading" color="primaryContrast">
-            {name()}
-          </TextView>
-          {accessToMilitaryInfo && (
-            <TextView textTransform="capitalize" variant="MobileBodyBold" color="primaryContrast">
-              {branch}
+    <Pressable {...pressableProps}>
```

Comment:
Should this still be a Pressable in the case where it can't be pressed? Any a11y concerns, that is, does the screen reader still read it as pressable?

Reply:
No I believe this should be fine based the conditionalized props. The reader would need to have a role assigned of button for it to know it should I think but I'll make sure to have QA confirm that.